### PR TITLE
Fixed spacing in model.mustache

### PR DESCRIPTION
**Pull request by kenisteward**
### PR checklist

- [x] Read the [contribution guidelines](https://github.com/swagger-api/swagger-codegen/blob/master/CONTRIBUTING.md).
- [x] Ran the shell script under `./bin/` to update Petstore sample so that CIs can verify the change. (For instance, only need to run `./bin/{LANG}-petstore.sh` and `./bin/security/{LANG}-petstore.sh` if updating the {LANG} (e.g. php, ruby, python, etc) code generator or {LANG} client's mustache templates). Windows batch files can be found in `.\bin\windows\`.
- [x] Filed the PR against the correct branch: `3.0.0` branch for changes related to OpenAPI spec 3.0. Default: `master`.
- [ ] Copied the [technical committee](https://github.com/swagger-api/swagger-codegen/#swagger-codegen-technical-committee) to review the pull request if your PR is targeting a particular programming langauge.

### Description of the PR

@sebastianhaas @wing328 

An extra line was being added in models.  See any an example in the PR here #6757 

Also amended typscript-angular2-all.bat and .sh to include angular 4 and angular 4.3 generation.

### CODE REVIEW

**Comment by kenisteward** [comment]
Note @wing328 I see we have run-all-petstore.cmd that's run by appveyor but they are multiple *-all.bat files.  This means tests are taking longer than needed due to double generating templates.   Should we just include *-all.bat and update the appveyor script or should we in the appveyor script just exclude once that have -all in them?

File: modules/swagger-codegen/src/main/resources/typescript-angular/modelGeneric.mustache [hunk 1]
```diff
@@ -10,7 +10,6 @@ export interface {{classname}} {{#parent}}extends {{{parent}}} {{/parent}}{
      */
     {{/description}}
     {{name}}{{^required}}?{{/required}}: {{#isEnum}}{{{datatypeWithEnum}}}{{/isEnum}}{{^isEnum}}{{{datatype}}}{{/isEnum}};
-
```
Comment (sebastianhaas) [inline modules/swagger-codegen/src/main/resources/typescript-angular/modelGeneric.mustache]: I like this

File: samples/client/petstore/typescript-angular-v2/default/model/ApiResponse.ts [hunk 2]
```diff
@@ -17,9 +17,8 @@
  */
 export interface ApiResponse {
     code?: number;
-
     type?: string;
-
     message?: string;
-
 }
+
```
Comment (sebastianhaas) [inline samples/client/petstore/typescript-angular-v2/default/model/ApiResponse.ts]: But do you know where these two lines come from? Why two?

Comment (kenisteward) [reply samples/client/petstore/typescript-angular-v2/default/model/ApiResponse.ts]: dangit.... I think I accidentally added another one after the simple eol.  I can go back and change that lol

Comment (kenisteward) [reply samples/client/petstore/typescript-angular-v2/default/model/ApiResponse.ts]: turns out it wasn't me.  It was there incorrectly from the enum generator

**Comment by wing328** [comment]
> Should we just include *-all.bat and update the appveyor script or should we in the appveyor script just exclude once that have -all in them?

@kenisteward maybe better to exclude *-all.bat

**Comment by wing328** [comment]
@kenisteward the PR looks good

cc @TiFu @taxpon @sebastianhaas @Vrolijkx

File: samples/client/petstore/typescript-angular-v2/default/model/ApiResponse.ts (hunk 2 above)
Comment (sebastianhaas) [reply samples/client/petstore/typescript-angular-v2/default/model/ApiResponse.ts]: @kenisteward ah I see. I'll create a PR to get rid of them eventually, unless anybody sees any value in having two empty lines at the end.

**Comment by kenisteward** [comment]
So in the model.mustache we include modelgeneric and enumgeneric. Whenever
there are nested mustaches it seems it takes all the end of file lines from
all of them.  Therefore if we have every mustache with end of file lines we
end up with more than we want in the output.  I've fixed the output by
simply adding one end line on the outer mustache and removing ones from the
inserted.

**Pull request merged**